src/index.html

```diff
@@ -11,7 +11,201 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="field">
+        <legend>
+          Personal information
+        </legend>
+        <div class="form-field">
+          <label for="Surname">
+            Surname:
+          </label>
+          <input
+            type="text"
+            name="Surname"
+            id="Surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+          </label>
+          <input
+            type="text"
+            name="Name"
+            id="name"
+            autocomplete="off"
+            required
+          >
```

Comment:
Do u need `id`s and `for`?
```suggestion
          <label>
            Name:
            <input
	            type="text"
	            name="Name"
	            autocomplete="off"
	            required
	          >
          </label>
          
```